pkg/kdump/kdump-view.jsx

```diff
@@ -41,15 +41,78 @@ import { Tooltip, TooltipPosition } from "@patternfly/react-core/dist/esm/compon
 import { OutlinedQuestionCircleIcon } from "@patternfly/react-icons";
 
 import { useDialogs, DialogsContext } from "dialogs.jsx";
+import { read_os_release } from "os-release.js";
 import { fmt_to_fragments } from 'utils.jsx';
 import { show_modal_dialog } from "cockpit-components-dialog.jsx";
 import { FormHelper } from "cockpit-components-form-helper";
 import { ModalError } from 'cockpit-components-inline-notification.jsx';
 import { PrivilegedButton } from "cockpit-components-privileged.jsx";
+import { ModificationsExportDialog } from "cockpit-components-modifications.jsx";
 
 const _ = cockpit.gettext;
 const DEFAULT_KDUMP_PATH = "/var/crash";
 
+const exportAnsibleTask = (settings, os_release) => {
+    const target = Object.keys(settings.targets)[0];
+    const targetSettings = settings.targets[target];
+    const kdump_core_collector = settings.core_collector;
+
+    let role_name = "linux-system-roles";
+    if (os_release.NAME === "RHEL" || os_release.ID_LIKE?.includes('rhel')) {
+        role_name = "rhel-system-roles";
+    }
+
+    let ansible = `
+---
+# Also available via https://galaxy.ansible.com/ui/standalone/roles/linux-system-roles/kdump/
+- name: install ${role_name}
+  package:
+    name: ${role_name}
+    state: present
+  delegate_to: 127.0.0.1
+  become: true
+- name: run kdump system role
+  include_role:
+    name: ${role_name}.kdump
+  vars:
+    kdump_path: ${targetSettings.path || DEFAULT_KDUMP_PATH}
+    kdump_core_collector: ${kdump_core_collector}`;
+
+    if (target === "ssh") {
+        // HACK: we should not have to specify kdump_ssh_user and kdump_ssh_user as it is in kdump_target.location
+        // https://github.com/linux-system-roles/kdump/issues/184
```

Comment:
This comment is outdated now, as we don't actually specify `.location` any more.

Reply:
I need to do a follow up anyway for self.createPackage, so I'll add it there.